span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGenerator.java

```diff
@@ -525,4 +570,14 @@ private void populateUrlParts(Request.Builder requestBuilder) {
   private static URL getNormalizedUrl(String url) throws MalformedURLException {
     return new URL(new URL(RELATIVE_URL_CONTEXT), url);
   }
+
+  private void maybeSetHttpUrlForOtelFormat(
+      Request.Builder requestBuilder,
+      Map<String, AttributeValue> attributeValueMap) {
```

Comment:
Shouldn't we using `Map<String, JaegerSpanInternalModel.KeyValue> tagsMap` here too?
There is an assumption that attribute map is filed with all tags key.

Reply:
yes, it should populated before the call.